roles/collect/meta/main.yml

```diff
@@ -1,51 +1,20 @@
 ---
 galaxy_info:
-  author: Chris Hambridge
-  description: Collect OCP usage data to send to Insights
-  company: Red Hat
-
-  # If the issue tracker for your role is not on github, uncomment the
-  # next line and provide a value
-  issue_tracker_url: https://github.com/project-koku/korekuta/issues
-
-  license: AGPLv3
-
-  min_ansible_version: 1.2
-
-  # If this a Container Enabled role, provide the minimum Ansible Container version.
-  # min_ansible_container_version:
-
-  # Optionally specify the branch Galaxy will use when accessing the GitHub
-  # repo for this role. During role install, if no tags are available,
-  # Galaxy will use this branch. During import Galaxy will access files on
-  # this branch. If Travis integration is configured, only notifications for this
-  # branch will be accepted. Otherwise, in all cases, the repo's default branch
-  # (usually master) will be used.
-  # github_branch:
-
-  #
-  # platforms is a list of platforms, and each platform has a name and a list of versions.
-  #
-  # platforms:
-  # - name: Fedora
-  #   versions:
-  #   - all
-  #   - 25
-  # - name: SomePlatform
-  #   versions:
-  #   - all
-  #   - 1.0
-  #   - 7
-  #   - 99.99
-
-  galaxy_tags: []
-  # List tags for your role here, one per line. A tag is a keyword that describes
-  # and categorizes the role. Users find roles by searching for tags. Be sure to
-  # remove the '[]' above, if you add tags to this list.
-  #
-  # NOTE: A tag is limited to a single word comprised of alphanumeric characters.
-  #       Maximum 20 tags per role.
-
+  author: 'Chris Hambridge'
+  description: 'Collect OCP usage data to send to Insights'
+  company: 'Red Hat, Inc.'
+  issue_tracker_url: 'https://github.com/project-koku/korekuta/issues'
+  license: 'AGPLv3'
+  min_ansible_version: 2.4
+  platforms:
+    - name: Fedora
+      versions:
+        - all
+    - name: CentOS
```

Comment:
I don't believe the Insights client runs/monitors non-RHEL systems (even if they are in the RHEL family).